api-report/pact-map.api.md

```diff
@@ -26,7 +26,7 @@ export interface IPactMap<T = unknown> extends ISharedObject<IPactMapEvents> {
 
 // @public
 export interface IPactMapEvents extends ISharedObjectEvents {
-    (event: "pending" | "accepted", listener: (key: string) => void): any;
+    (event: "pending" | "accepted", listener: (key: string, sequenceNumber: number) => void): any;
```

Comment:
One thing to think about exposing this as an event param - the information vanishes after the event fires.  Would we want to support inspecting an already-accepted value to learn when it was accepted?

In particular I'm thinking about this scenario:
1. Acceptance happens at sequence number 52
2. v1 summarizer client summarizes at sequence number 52
3. Everyone crashes
4. New client loads, using the summary at sequence number 52

That new client could take it as implicit that 52 must have been the sequence number of acceptance (since the pact map value is accepted within the snapshot) but it might be more straightforward if they could retrieve that number on demand.

Reply:
> Would we want to support inspecting an already-accepted value to learn when it was accepted?

@ChumpChief, I was a bit more hesitant since I thought inspecting the sequence number at the time of acceptance would be a niche use case. However, the scenario you presented is an issue without it. We can modify `PactMap.get()` to return the entire `IAcceptedPact` (or add a separate API for it), but I think we should only do so we expect this to be a more common use case or if we plan on expanding `IAcceptedPact` in the future. Otherwise, I think want store the emitted sequence number in a more resilient way within the same-container example.

Reply:
@ChumpChief, I updated the PR to use a shared map to store the accepted sequence number. This should handle the above scenario, but obviously adds the overhead of introducing a shared map that may not be used for anything else. Let me know what you think/if you want to sync up to compare the options